sys/src/9/port/devufs.c

```diff
@@ -167,7 +186,7 @@ ufswrite(Chan *c, void *a, int32_t n, int64_t offset)
 
 	switch (QID(c->qid)) {
 	case Qmount:
-		mount((char*)a);
+		mount((char*)a, n);
```

Comment:
The general way to do this is set n to desired value (or don't set n) and then break; and fall out of the switch and exit.